t/08-autotest.t

```diff
@@ -449,12 +449,9 @@ subtest make_snapshot => sub {
 };
 
 subtest loadtestdir => sub {
-    use lib 't/data/tests';
-    $bmwqemu::vars{CASEDIR} = 't/data/tests';
+    $bmwqemu::vars{CASEDIR} = '.';
```

Comment:
that might work to avoid the warning in this test, but it's not a real world scenario.
I think loadtestdir needs to be fixed.

Reply:
It's confusing to me.  :confused:
I am closing the PR and let's file a Progress ticket for improving `loadtestdir()` subroutine

Reply:
I created https://progress.opensuse.org/issues/175099